core/src/client/async_client/mod.rs

```diff
@@ -115,6 +117,23 @@ impl ClientBuilder {
 		self
 	}
 
+	/// Set the interval at which pings frames are submitted (disabled by default).
+	///
+	/// Periodically submitting pings at a defined interval has mainly two benefits:
+	///  - Directly, it acts as a "keep-alive" alternative in the WebSocket world.
+	///  - Indirectly by inspecting trace logs, ensures that the endpoint is still responding to messages.
```

Comment:
I'm not super excited about having the pongs to be logged as `trace` because the and trace` logs are so damn verbose.

I think the `pongs` should be `info` or `debug`, but I'll tidy up the logs in another PR.